packages/strapi/lib/middlewares/parser/index.js

```diff
@@ -1,11 +1,49 @@
 'use strict';
 
 const body = require('koa-body');
-const qs = require('koa-qs');
+//const qs = require('koa-qs');
+var merge = require('merge-descriptors');
+const qs = require('qs');
 
 /**
  * Body parser hook
  */
+
+const addParser = app => {
+  merge(app.request, {
+    /**
+     * Get parsed query-string.
+     *
+     * @return {Object}
+     * @api public
+     */
+    get query() {
+      var str = this.querystring;
```

Comment:
please use const and let instead of vars. Also make sure to use variables that have some meaning :) (cache on c for example)

Reply:
code cleaned and avoided vars.